src/components/Dao/DaoContainer.tsx

```diff
@@ -104,10 +111,14 @@ class DaoContainer extends React.Component<IProps, null> {
               render={this.daoProposalRoute}
             />
 
+            <Route exact path="/dao/:daoAvatarAddress/crx/proposal/:proposalId"
```

Comment:
As thi sis probably best thought of as an alternative view of the proposal, perhaps the url can be:

`"/dao/:daoAvatarAddress/proposal/:proposalId/competition"`